code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -38,7 +38,8 @@
 	if(part)
 		part.implants -= src
 		part = null
-	forceMove(get_turf(imp_in))
+	if(!gcDestroyed)
+		forceMove(get_turf(imp_in))
```

Comment:
You don't *have* to check this since `qdel()` already does and thus it would only happen once

Reply:
I don't understand.

Reply:
Oh, you mean I don't have to check `if(!gcDestroyed)` because `Destroy()` isn't called if `gcDestroyed` is set. However this proc may be called by something other than `Destroy()`. Does it not make sense to keep the check?

Reply:
The check doesn't break anything, it's just not necessary to prevent the infinite loop. You can keep it around anyway if you want to.